app/controllers/AccountManagement.scala

```diff
@@ -97,7 +97,7 @@ object ManageDelivery extends ContextLogging {
       _ <- EitherT(tpBackend.suspensionService.renewIfNeeded(sub,newHoliday))
       // 26 because one year from now could be end of second years sub + 2 extra months needed in calculation to cover setting a 6-week suspension on the day before your 12th billing day!
       oldBS <- EitherT(tpBackend.commonPaymentService.flatMap(_.billingSchedule(sub.id, account, numberOfBills = 26).map(_ \/> "Error getting billing schedule")))
-      result <- EitherT(tpBackend.suspensionService.addHoliday(newHoliday, oldBS, account.billCycleDay, sub.termEndDate)).leftMap(getAndLogRefundError(_).map(_.code).list.mkString(","))
+      result <- EitherT(tpBackend.suspensionService.addHoliday(newHoliday, oldBS, account.billCycleDay, sub.termEndDate)).leftMap(getAndLogRefundError(_).map(_.code).list.toList.mkString(","))
```

Comment:
scalaz kindly returns an IList (invariant list) instead of a real list, so you have to toList if you want a normal list.